<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dual Camera Recorder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:disabled {
	color: gray
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="Camera" name="cam1" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="Camera" name="cam2" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonStartRecording">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Start Recording</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButtonStopRecording">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Stop Recording</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionShowCrosshair"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionTakeSnapshot"/>
    <addaction name="actionOpenRecordings"/>
   </widget>
   <addaction name="menuImage"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShowCrosshair">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show crosshair</string>
   </property>
  </action>
  <action name="actionTakeSnapshot">
   <property name="text">
    <string>Take snapshot</string>
   </property>
  </action>
  <action name="actionOpenRecordings">
   <property name="text">
    <string>Open recordings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Camera</class>
   <extends>QWidget</extends>
   <header>camera.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
